src/app/(app)/attendance/mark/page.tsx

```diff
@@ -1,14 +1,21 @@
-import React from 'react'
```

Comment:
Wondering if we can rename this route to `/review`

Reply:
I have renamed this route